pkg/standalone/standalone.go

```diff
@@ -1084,7 +1115,7 @@ func checkFallbackImgForPlacement(imageInfo daprImageInfo, fromDir string) bool
 func resolveImageURI(imageInfo daprImageInfo) (string, error) {
 	if imageInfo.imageRegistryURL != "" {
```

Comment:
```suggestion
	if len(strings.TrimSpace(imageRegistryURL)) != 0 {
```

If we are using TrimSpace everywhere, it should be here as well.